test/test_prototype_transforms_functional.py

```diff
@@ -131,7 +122,7 @@ def test_scripted_vs_eager(self, test_id, info, args_kwargs, device):
             actual,
             expected,
             **info.get_closeness_kwargs(test_id, dtype=input.dtype, device=input.device),
-            msg=parametrized_error_message(*other_args, *kwargs),
+            msg=parametrized_error_message(*other_args, **kwargs),
```

Comment:
I botched this when I introduced the extra information. Without the `**` we would only include the names of the parameters rather than the name-value-pair.